spark/v3.0/build.gradle

```diff
@@ -129,7 +129,6 @@ project(":iceberg-spark:iceberg-spark3-extensions") {
     compileOnly project(':iceberg-data')
     compileOnly project(':iceberg-orc')
     compileOnly project(':iceberg-common')
-    compileOnly project(':iceberg-spark')
```

Comment:
This and the ones below are not necessary. I removed them so they are not consistent with the build.gradle for v.31 and v3.2.

Reply:
Sounds good to me.